easybuild/easyconfigs/g/GCC/GCC-system.eb

```diff
@@ -0,0 +1,19 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+# License::   3-clause BSD
+##
+
+easyblock = 'SystemCompiler'
+
+name = "GCC"
+version = "system"
+
+homepage = "<will be defined by the SystemCompiler easyblock>"
+description = ""   # will be defined by the SystemCompiler easyblock when left empty or provide a custom description
```

Comment:
I feel it's better to fill these in here, like they are for the other GCC easyconfigs?

It doesn't make much sense that the easyblock is taking care of this, since we already know which compiler will be used (I'm not sure anymore what the motivation behind this was)...